src/challenge-2/challenge-2.c

```diff
@@ -76,7 +74,7 @@ void follower_event_loop()
 
 void leader_event_loop()
 {
-    Queue* queue = queue_init(5);
+    Queue* queue = queue_init(5, queue_json_object_free);
```

Comment:
Optional: Might be worth renaming this `conch_request_queue` just to disambiguate what queue this is (given we've got queues all over the place now).

Reply:
Done!